app/models/user.rb

```diff
@@ -10,6 +10,8 @@ class User < ApplicationRecord
   validates :name, presence: true
   validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
 
+  enum role: %i[user admin]
```

Comment:
- [ ] Your `role` in the database is of the type `string`, but here you are declaring `enum` for it. Typically, enums use `integer` column types. As a result, I am unable to make a user an "admin". Kindly remove this enum line, or update the database column to use the correct type